.github/workflows/rust.yml

```diff
@@ -2,9 +2,9 @@ name: Rust
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, dev ]
```

Comment:
Yes, the automated github checks finally run! I wonder why this was necessary, as these changes should already be in the `dev` branch, so it should also already be in the other two pull requests